spec/features/availability_spec.rb

```diff
@@ -18,11 +17,63 @@
     end
   end
 
+  describe 'Holdings for SCSB records' do
+    context 'when a record has no format' do
+      it 'still displays the record with a ReCAP location' do
+        stub_holding_locations
+        visit 'catalog/SCSB-7935196'
+        expect(page).to have_selector '.library-location', text: 'ReCAP'
+      end
+    end
+  end
+
+  # There are issues with correctly setting the user agent with chromedriver
+  context 'when visited by an indexing bot', driver: :iphone do
+    before do
+      page.driver.add_headers('User-Agent' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1 (compatible; AdsBot-Google-Mobile; +http://www.google.com/mobile/adsbot.html)')
+    end
+
+    describe 'viewing a catalog record', js: true do
+      before do
+        stub_holding_locations
+        visit '/catalog/3256177'
+      end
+
+      xit 'does not render the holding section' do
+        expect(page).not_to have_selector('.availability--holding')
+      end
+    end
+
+    describe 'viewing a record for an electronic holding', js: true do
+      before do
+        stub_holding_locations
+        visit '/catalog/857469'
+      end
+
+      xit 'does not render the online section' do
+        expect(page).not_to have_selector('.availability--online')
+      end
```

Comment:
I think this should change to `expect(page).to have_selector('.availability--online')` since there is an online section for this record through google books.

Reply:
These tests are skipped because of the comment on line 30. Our PR did not break this. We are not sure why this context is testing 'bots' with an iphone driver. Created a follow up ticket: https://github.com/pulibrary/orangelight/issues/2396